grails-app/assets/javascripts/streama/streama.translations.js

```diff
@@ -10,5 +10,5 @@ angular.module('streama.translations', ['pascalprecht.translate'])
 	})
 
 	.run(function ($rootScope) {
-		$rootScope.availableLanguages = ['cn', 'en', 'fr', 'es', 'de', 'kr', 'nl', 'pt', 'da', 'ja', 'it', 'ar', 'ru', 'hu'];
+		$rootScope.availableLanguages = ['en', 'fr', 'es', 'de', 'kr', 'nl', 'pt', 'da', 'ja', 'it', 'ar', 'ru', 'cn', 'hu'];
```

Comment:
what exactly does this fix?

Reply:
I don’t remember why I made this change because it’s too much time
But what I do remember is that there was a problem only because of this arrangement and I think when trying to change a language